src/Products/SiteAccess/tests/testVirtualHostMonster.py

```diff
@@ -139,6 +139,48 @@ def test(self, vaddr=vaddr, vr=vr, _vh=_vh, p=p, ubase=ubase):
 
     setattr(VHMRegressions, 'testTraverse%s' % i, test)
 
+class VHMPort(unittest.TestCase):
+
+    def setUp(self):
+        import transaction
+        from Testing.makerequest import makerequest
+        from Testing.ZopeTestCase.ZopeLite import app
+        transaction.begin()
+        self.app = makerequest(app())
+        if 'virtual_hosting' not in  self.app.objectIds():
+            # If ZopeLite was imported, we have no default virtual
+            # host monster
+            from Products.SiteAccess.VirtualHostMonster \
+                import manage_addVirtualHostMonster
+            manage_addVirtualHostMonster(self.app, 'virtual_hosting')
+        self.app.manage_addFolder('folder')
+        self.app.folder.manage_addDTMLMethod('doc', '')
+        self.app.REQUEST.set('PARENTS', [self.app])
+        self.traverse = self.app.REQUEST.traverse
+
+    def tearDown(self):
+        import transaction
+        transaction.abort()
+        self.app._p_jar.close()
+
+    def testPassedPort(self):
```

Comment:
Can you add `IPV4` to the name here, for parity / explicitness?

Reply:
Done.